lib/wellies_web/controllers/weather_controller.ex

```diff
@@ -1,8 +1,14 @@
 defmodule WelliesWeb.WeatherController do
   use WelliesWeb, :controller
+  alias WelliesWeb.CurrentTemperature
+  alias WelliesWeb.FiveDaysTemperature
 
-  def weather(conn, %{"location" => location} = _params) do
-    render(conn, "weather.html", location: location)
+  def weather(conn, %{"location_id" => location_id} = _params) do
+    now = CurrentTemperature.request_current_forecast(location_id)
+    five_days = FiveDaysTemperature.request_5d_forecast(location_id)
```

Comment:
I think the API would sound much cleaner if it was something in the lines of `FiveDaysTemperature.forecast(location_id)`. I would even expect it to be something like `FiveDaysForecast.request(location_id)`, but that's more of an opinion I guess

Reply:
Also, what's the `five_days` variable used for?